src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -194,10 +195,11 @@ def nwm_output_generator(
 
     if test:
         flowveldepth.to_pickle(Path(test))
+        import pdb; pdb.set_trace()
```

Comment:
Forgot to delete this line. I'll delete it

Reply:
Looks like it's still there?